tox.ini

```
@@ -1,9 +1,9 @@
 [tox]
 envlist =
-    py{36,37,38}-test{,-alldeps,-devdeps}{,-cov}
-    py{36,37,38}-test-numpy{116,117,118}
+    py{36,37,38}-test{,-all}{,-dev,-latest,-oldest}{,-cov}
+    py{36,37,38}-test-numpy{116,117,118,119}
     py{36,37,38}-test-scipy{12,13,14,15}
-    py{36,37,38}-test-astropy{30,40,lts}
+    py{36,37,38}-test-astropy{32,40,lts}
```

```suggestion
    py{36,37,38}-test-astropy{32,40}
```
Maybe only need astropy40 since astropylts resolves to the same?

---

That all comes from the package template, where I would guess that astropy will let `40` evolve and keep `lts` pinned to 4.0. Anyway, if we adopt the oldest-latest testing strategy, we need pinned versions between those, and don't have to single out `lts`, and it can go.

---

Ah OK then I don't mind removing it or sticking to the template